syntax = "proto3";

package WeldBeadModel;

message InputVector {
  repeated double data = 210 [packed=true];
}

message OutputVector {
  repeated double data = 216 [packed=true];
}

service InferenceService {
  rpc inference (InputVector) returns (OutputVector) {}
}

